### PR TITLE
Upgrade to 0 15 version

### PR DESCRIPTION
**Pull request by isaacmg410**
This PR, can't be merged until #19 will be merged into master. 

Related with #32 

- [x]  Upgrade version Decidim to 0.15.2
- [x]  rails decidim:upgrade to add missing migrations
- [x]  update the overwritten files
- [x] add delayed_job
- [x] add whenever
- [x] documentation

### CODE REVIEW

**Comment by isaacmg410** [comment]
@tramuntanal @aitorlb can you review this PR please?

File: docs/HOW_TO_DEPLOY_0_15.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# KEEP IN MIND WHEN DEPLOYING 0.15
+
+Take a look to the Changelog https://github.com/decidim/decidim/blob/0.15-stable/CHANGELOG.md to specific actions. Instead of run the "Upgrade notes" on Changelog 0.15, execute next queries to prevent some crashes"
+
+## Notes.
+- Comment how the block system of home works. Slider must be activated and FEDER, 0.14
+- At 0.15 we can say how many participatory spaces come out in each block of the home. By default they are 4.
+
+- DelayedJOB has been installed, to run metrics. `bin/delayed_job start` must be executed on production
```
Comment (tramuntanal) [inline docs/HOW_TO_DEPLOY_0_15.md]: We must configure capistrano to do so, we should not execute this manually...

File: docs/HOW_TO_DEPLOY_0_15.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# KEEP IN MIND WHEN DEPLOYING 0.15
+
+Take a look to the Changelog https://github.com/decidim/decidim/blob/0.15-stable/CHANGELOG.md to specific actions. Instead of run the "Upgrade notes" on Changelog 0.15, execute next queries to prevent some crashes"
```
Comment (tramuntanal) [inline docs/HOW_TO_DEPLOY_0_15.md]: I don't understand this paragraph. You say to check the changelog but then you say to ignore it and instead run "next queries", but where are the "next queries"?

File: docs/HOW_TO_DEPLOY_0_15.md (hunk 1 above)
Comment (isaacmg410) [reply docs/HOW_TO_DEPLOY_0_15.md]: We are not using capistrano in this project. So.. we need to do this manually, no?

File: docs/HOW_TO_DEPLOY_0_15.md (hunk 2 above)
Comment (isaacmg410) [reply docs/HOW_TO_DEPLOY_0_15.md]: Ups, sorry! It was a copy paste from another document... It's fixed now

File: app/assets/stylesheets/decidim.scss [hunk 3]
```diff
@@ -604,12 +604,12 @@ h5 {
...
 	&.hollow{
 		border-color: $RedGenCat !important;
-
-		color: $RedGenCat !important;
+		color: $white !important;
 	}
 }
 .button{
 	&.hollow{
+		color: $white !important;;
```
[the remark this answers is not in the document]
Comment (isaacmg410) [reply app/assets/stylesheets/decidim.scss]: Ups,  my mistake.
Fixed!

**Pull request merged**